src/components/Questionnaire/QuestionTypes/MedicationRequestQuestion.tsx

```diff
@@ -778,6 +779,32 @@ const MedicationRequestGridRow: React.FC<MedicationRequestGridRowProps> = ({
   const isReadOnly = !!medication.id;
   const { hasError } = useFieldError(questionId, errors, index);
 
+  const [currentInstructions, setCurrentInstructions] = useState<Code[]>(
+    dosageInstruction?.additional_instruction || [],
+  );
+
+  const updateInstructions = (instructions: Code[]) => {
+    setCurrentInstructions(instructions);
+    handleUpdateDosageInstruction({
+      additional_instruction:
+        instructions.length > 0 ? instructions : undefined,
+    });
+  };
+
+  const addInstruction = (instruction: Code) => {
+    if (!currentInstructions.some((item) => item.code === instruction.code)) {
+      updateInstructions([...currentInstructions, instruction]);
+    } else {
+      toast.error(`${instruction.display} ${t("is_already_selected")}`);
```

Comment:
```suggestion
      toast.warning(`${instruction.display} ${t("is_already_selected")}`);
```

I think this should be a warning instead of error because it is not an error

what do you think ?

Reply:
Yes